backend/src/yeomanui.ts

```diff
@@ -116,6 +116,24 @@ export class YeomanUI {
 		await this.rpc.invoke("updateGeneratorsPrompt", [generators.questions]);
 	}
 
+	public async _notifyGeneratorsInstall(args: any[], force: boolean) {
+		if (!_.isNil(args)) {
+			const isGeneratorsPrompt: boolean = await this.rpc.invoke("isGeneratorsPrompt");
+			if (isGeneratorsPrompt || force) {
+				const message = this.getGeneratorsInstallingMessage(args);
+				this.youiEvents.getAppWizard().showInformation(message, MessageType.prompt);
+			}
+		}
+		this.uiOptions.installGens = (_.isObject(args) && _.isEmpty(args)) ? undefined : args;
```

Comment:
can you explain, what is expected result here ? _.isObject() is true on arrays, functions, objects, regexes, new Number(0), and new String('')